assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -86,12 +92,39 @@ const initial = {
     }),
     toggle: () => ({ isOpen }) => ({ isOpen: !isOpen }),
     close: () => () => ({ isOpen: false }),
+    fetchReadStart: () => () => ({ loading: true }),
+    fetchReadSuccess: (payload) => () => ({
+      ...payload,
+      loading: false,
+    }),
+    fetchReadError: () => () => ({
+      fetchError: true,
+      loading: false,
+    }),
+    read: (key) => ({ read }) => ({
+      // optimistically update the notification as "read"
+      read: read.indexOf(key) === -1 ? read.concat(key) : read,
+      updating: true,
+    }),
```

Comment:
This makes me feel like `state.read` should be an object. I feel like that's what's coming back from the endpoint anyway and make this just:
```
  readTip: (key) => ({ readTips }) => ({
      readTips: {
        ...readTips,
        [key]: true
      },
  }),
```

Also I know one is the action read and the other is the past tense read, but it makes it a little weird to read. Might be nice to name this something a little different like `setTipAsRead` or `readTip`

Reply:
I also don't think We should add the `updating` state here. persisting our local state is a separate action than updating our local state. I'm also not sure if we need to keep track of whether or not our persisting call was successful in here

Reply:
I now update the `readTips` state using `deriveState`

Reply:
Oh nice! good call